headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,94 @@
+#ifndef HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+#define HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+
+struct LiveActor {
+    uint16_t id;     /* The id of the Actor in the actor list */
+    uint16_t kind;   /* The specie of the Actor */
+    bool is_enabled; /* true when the actor is loaded, false otherwise (should be checked if unsure)
+                      */
+    undefined field3_0x5;
+    uint16_t hanger;
+    uint8_t sector;
```

Comment:
Do we know what these fields are? Comments would be nice if possible.

Reply:
I think that sector is related to whether they are loaded of not in a script, but I never used something different than the default. They can easily be edited in SkyTemple, and named like that. I'll ask.

Reply:
It's okay, if you don't know we don't have to bother right now. We can always come back and add comments if someone needs them.